### ISSUE TITLE
Restaurant District: Disrupting the Restaurant Biz

### ISSUE THREAD

**Issue by fastdraw07** [district0x/district-proposals#28]
# The Restaurant Business, Decentralized (I.e. Disrupting the restaurant # industry)

What if we could decentralize the entire restaurant business, where anyone could run a restaurant out of their home through the blockchain, with no government licensing needed? A peer-to-peer restaurant solution, managed and facilitated by the blockchain, lowering costs and increasing supply and choices. At the Restaurant District, anyone could run a restaurant out of their home, or they could rent out a location, like a regular restaurant. Within the Restaurant District, people could BBQ steaks on their front porch, set out tables on their front lawn, and serve people at night to make extra cash, or they could do it full-time. Anyone in the district could open and close their restaurant at any time, serve what they want, and charge what they want. And, what restaurants are serving, will be listed for any local citizens to view. Cut out the middleman, the high cost of managing a traditional restaurant, and the choices for what's to eat in town, just got a whole lot more interesting. The Restaurant District would turn the business of restaurateuring into a real community event.

## Reputation and Price

the blockchain. They get a world-class meal, served in a friendly environment. Some restaurant owners could just have a set price if they choose, but it should be up to them; It's their restaurant. It's their business. That's the miracle of blockchain and decentralization. Kids serving snowcones or lemonade in the summer to help pay for college? No problem, it's all facilitated on A restaurant's reputation will grow with the more happy customers they serve. And, their reputation will be there for all to see. Top chefs could host dinners at their nice homes, and a bidding process could determine who gets to eat that night. A top chef could rake in the cash, especially during big events like the Superbowl or the World Cup Finals, a bidding system could set the price. The top bidders would show up, pay, and the transaction would be recorded with blockchain technology.

## Insurance

Reputation is really helpful, but insurance against food poisoning or bodily harm to others (e.g. incase of a fire?) could be displayed on a restaurant's profile to give increased confidence to customers. If the insurance is purchased through the blockchain, say through, Etherisc, then all the better.

## Cleanliness Checklist

A cleanliness checklist could be provided for restaurant owners to submit to show that they are ready and clean for business. Customers could provide feedback saying if the cleanliness checklist submission was honest, in their opinion. This feedback could hurt or help the restaurant owner's reputation, and they would have an economic incentive, to be honest with the checklist.

## Delivery

Restaurants owners have the option of delivery as an extra charge, or exclusively, if they don't want people coming to their homes. The app will have map capabilities like any delivery app would.

## Putting the Kids to Work

Certain neighborhood, Peer-to-peer restaurants, could have a big, cost-saving advantage: Kids. Parents could do the cooking, set the price, and all the adult stuff, while the kids help clean and serve. And, parents don't have to pay their kids minimum wage!

## Community

The community could vote for the best restaurants on a monthly basis. Categorized awards could be given out and automated payouts could be given to restaurants who serve their customers the best.

## Opportunity

With no direct blockchain startup competitor to the Restaurant District, this idea of disrupting the restaurant business through decentralization is an excellent opportunity for district0x. The Restaurant District would be newsworthy, valuable on both, the business and consumer side, and would get people excited about the district0x.

## Initial Restaurant Features

- General profile description
- Picture upload of seating area and staff (optional)
- Menu item listing
- Menu item pricing
- Hours/days of operation
- Set location of the restaurant if allowing in-home dining. (optional)
- Set max # of seating (If location is set)
- Set first come, first serve, or reserve seating only. (If location is set)
- Options for delivery: Price, Radius
- Rate customers
- Set how many seats are currently taken. Customers will see this as Seats Available.

## Initial Customer Features

- Searchable and browseable menu items
- Filter by radius
- Purchase delivery
- Reserve seating (Funds in escrow)
- Restaurant rating
- Map integration for locating restaurants
- Ability to leave a tip

## Use Cases

- Sally, from Louisiana, opens her Jambalaya Delivery Service. She sells her delicious Jambalaya in small, medium, or large pots. Her sons do the driving, and they do a great job, getting the Cajun food there, hot and fast.
- Award-winning chef closes his restaurant and opens his own in-home restaurant within district0x, serving his delectable creations only twice a week, using a bidding process. He attracts the most affluent of the city, making as much in one night, as he made in one week at his restaurant, working for his old boss. He has an 80-inch UltraK television he uses to show the big events, where he typically makes a fortune in one night. 
- Pizza Master Joe goes to school during the day and makes the most wonderful pizza using his uncle's secret sauce at night. The neighborhood kid gets a cut, delivering around the neighborhood, using his bicycle.

**Comment by lkngtn** [comment]
I really like the idea, though I'm a bit concerned that because I think this is illegal in my jurisdiction (US), it would be a bit risky to participate. Since you have to publicly post your address/identity in order for customers to find you, it seems like it would be difficult for this to be hugely successful unless there was some assurance that listing your "restraunt" on the site wouldn't get you in trouble.

**Comment by tfg615** [comment]
Agree with @lkngtn that the legality of this concept (restaurant licenses + child labor laws) could hinder this project. However, with some more build out in order to ensure the legal components are accounted for I think there is some potential here. Conceptually I love it, but from a real world perspective it needs some more detail and build out. -tfg_50

**Comment by fastdraw07** [comment]
I think if district0x worries too much about licensing and regulation, they'll lose out to other startups who are more bold. Blockchain already got legalized in New Hampshire and Nevada. That's just a start. The benefits of blockchain and peer-to-peer are so obvious that many politicians may be afraid to deny it. 

As far as the child labor laws, that is a great point. As far as I know, "Chores" are not illegal. Parents should be able to raise their kids anyway they want, unless they are hurting them. Putting them to work at their own residence is not illegal as far as I know. Parents allocate chores to their children all the time. There is no legal precedent that I know for making chores against the law.

**Comment by tfg615** [comment]
To be clear @fastdraw07, I'm not saying that this concept is not possible. I think there just must be some more build out to ensure district users of a concept like this are not subject to fines because they were not made of aware of such licensing, food prep, etc. laws.

**Comment by lkngtn** [comment]
> I think if district0x worries too much about licensing and regulation, they'll lose out to other startups who are more bold. Blockchain already got legalized in New Hampshire and Nevada. 

I think the concept of DAOs means that new types of businesses that may not have been feasible before will be feasible now, simply because there is no way for a government to feasibly take it down. I think we will se a lot of regulatory change over the next few years as governments start to adapt to that new reality and need to balance what they actually want to do with what they can pragmatically do. It will be really interesting to watch that develop. 

However, I think things like this, especially since participants would not be anonymous, would be a significant risk to participating (not necessarily to hosting the site since it's on the blockchain) and that would limit the amount of participants until there is more of a regulatory framework them to participate in. Running a real restaurant is already incredibly risky, add on top of the fact that you are doing so illegally and I think that makes it a difficult proposition. However, I do know that there are some one-off type "event" listing services that have a similar function and perhaps modeling this after that might be effective. 

For example, https://www.bonappetour.com/, allows people to host specific dinner parties as one-offs rather than running a restaurant specifically. That seems to be working okay, atleast, they have bee around for a few years.

**Comment by fastdraw07** [comment]
@lkngtn Actually, I think since the participants would not be anonymous, that local governments would be more open to allowing this. Since, the customers know "where you live", I think that puts a check on behavior of the providers. An employee at a restaurant is more anonymous than a provider at a home. It's the anonymous nature of the economy that government's don't like. That's my opinion. 

I think the app is much less risky than AirBnB, and I will outline that in the proposal. 

The app could be rolled out in cities that would be willing to try it out. If you believe in the blockchain, then you know it will work, and there should be no worries. 

Thanks for that link. I'll probably include it in the proposal, which is still a work in progress.

**Comment by fastdraw07** [comment]
@tfg615 I'm not doubting your commitment to the blockchain. I understand the concerns. There isn't a single blockchain project out there that can't come under the scrutiny of government.

**Comment by 8lcarte** [comment]
As someone who's opened a restaurants before, I believe the food handling and license issues would be huge. However, my hope would be that most of those issues could all be streamlined by building great software with easy onboarding procedures (like driving for Uber). Ideally you could have some bounties for people who go around helping people get into compliance and "random inspector" bounties. My hope is that by building this community as a district we could leverage the crowd to help assuage these issue. 

I like your idea of a cleanliness checkllist but it must match the checklist of a standard food handling checklist. A professional chef may be okay with maintaining those standards but I wonder if the average household would be (not a criticism I just don't really know).

I really love the idea I just know most cities/states get squirmy when it comes to food regulation.

**Comment by lkngtn** [comment]
> Actually, I think since the participants would not be anonymous, that local governments would be more open to allowing this. Since, the customers know "where you live", I think that puts a check on behavior of the providers. An employee at a restaurant is more anonymous than a provider at a home. It's the anonymous nature of the economy that government's don't like. That's my opinion.

I agree that because users are not anonymous the government would be more likely to work with the project to ensure that its users follow health and food safety regulations, however, if the intended purpose of the site is to reduce the barrier of entry for creating a restaurant by getting around those regulations, that seems less likely. As others have pointed out, there would need to be some effort put in place to work with local jurisdictions and provide help for participants to follow whatever regulations are required. 

That will create significant overhead for the project which would need to be covered by some sort of fees, and I wonder if there is a central authority that collects those fees in order to provide that service how much benefit there is for this to be done on the blockchain in the first place? 

Taking a step back, if we assume that the goal of the project is to reduce the amount of regulatory oversight by making a platform for people to serve food out of their home that is difficult for regulators to shut down, we need to find some other way to minimize risk to participants. The idea of not running restaurants but instead coordinating scheduled meal events for small parties might work because that would be more likely to be seen as a private gathering rather than a typical restaurant. 

Aspiring restaurant owners could test their recipes and menus up front, build up a following, and then if they are successful with that low-risk endeavor they could then transition their service into a full-on restaurant that works within the existing health and safety regulations of their jurisdiction.

**Comment by fastdraw07** [comment]
@lkngtn I think both approaches could be done simultaneously.

But, If a bounty program was offered by district0x for going to local governments and getting approval for DApps like the Restaurant District as a pure P2P service, then that could be more efficient than a blanket approach to government regulatory burdens, since each locality is different. Once a locality has been proved, then users will know they can operate, legally. Outside of those approved localities, users will have to take the risk if they want to operate.

**Comment by 8lcarte** [comment]
The reason for it to be decentralized is that you are creating a whole new way to make sure food providers are licensed correctly that cities & consumers alike could track. By definition a centralized service could not create a totally trustless method for creating lower barrier to entry micro-restaurant creation. If you can find particular municipalities looking for tech like this to stimulate a growing restaurant industry I think you may be onto something.

On the consumer side you'd also then have a totally curated "micro-restaurant" list that cannot be influenced by a central system as is alleged of yelp. 

Regarding @lkngtn's comment on significant overhead covered by fees. I'm not sure why a district like this couldn't collect fees and provide the service the same way a central authority would. Obivously the business model for a centralized business is easier to create as this is what most businesses are today but with well designed bounty programs, and a system for voting on budget allocation, etc I think you could run the whole thing as one district. Allowing the crowd to participate in this type of thing could align the government bodies you are trying to pitch on the regulation even more with you.

**Comment by lkngtn** [comment]
I totally agree this could be done in a decentralized fashion, and I'm actually hopeful that that will happen, just from a pragmatic approach a lot of the infrastructure is still being built to support that. While that is being built approaches that can be successful purely on the economics of a decentralized marketplace without as much need for supporting services seem like they would be more likely to succeed.

I completely agree that ultimately that initial approach could evolve into something far more significant, and I welcome that. My suggestion is more a tactical go-to-market strategy rather than a critique of the vision.

**Comment by tfg615** [comment]
"Aspiring restaurant owners could test their recipes and menus up front, build up a following, and then if they are successful with that low-risk endeavor they could then transition their service into a full-on restaurant that works within the existing health and safety regulations of their jurisdiction."

Love that idea @lkngtn 

This is all great feedback and exactly what the team is hoping for in the district proposals. Keep it coming on this project, and all the other great ones currently proposed.

**Comment by fastdraw07** [comment]
I'm just hopeful that the much of the decentralized DApps that allow us to transact with each other will be built before fiat currencies start collapsing.

**Comment by EndymionJkb** [comment]
I just got pitched a very similar idea (for a traditional centralized app)! The regulatory hurdles are 
well nigh insurmountable in most jurisdictions, as others have noted. The profit margin for home chefs would also be low: they don't enjoy the economies of scale restaurants have, since they need to buy retail for small portions. This would also 

There are services like Blue Apron that deliver *ingredients* and instructions so that people can make their own food. And there are services like Bumble BFF that encourage "flash social gatherings" of compatible strangers who happen to be in the same neighborhood but otherwise wouldn't meet. 

Perhaps "Foodie Bumble," or "Bumble Potluck" where people bring their own food to spontaneous gatherings? Or a "Social Yelp" that helps people meet at restaurants who'd otherwise be dining alone (e.g., business travelers). There are probably variants of the main idea that avoid the legal issues.

Uber did something "illegal" and got away with it because the value of the service is just so high, and so obviously better than monopolized, essentially state-run taxi services. In contrast, I don't think the existing food provision landscape is so bad that everyone would immediately adopt this!

https://www.fastcompany.com/3061498/the-food-sharing-economy-is-delicious-and-illegal-will-it-survive
http://www.shareable.net/blog/the-shareable-food-movement-meets-the-law
http://www.shareable.net/blog/job-creating-underground-food-market-shut-down

**Comment by fastdraw07** [comment]
@EndymionJkb Here's one (Not decentralized) billed as AirBnB for dinner: https://techcrunch.com/2014/11/25/feastly-seed-funding-westly-group/

**Comment by EndymionJkb** [comment]
There are a lot! eatwith.com, mealsharing.com, josephine.com,
cassaroleclub.com, karmakitchen.org, leftoverswap (app), foodsharing.de

On Jul 14, 2017 11:49 AM, "fastdraw07" <notifications@github.com> wrote:

@EndymionJkb <https://github.com/endymionjkb> Here's one (Not
decentralized) billed as AirBnB for dinner: https://techcrunch.com/2014/11
/25/feastly-seed-funding-westly-group/

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/district0x/district-proposals/issues/28#issuecomment-315394619>,
or mute the thread

**Comment by fastdraw07** [comment]
@EndymionJkb Any on the blockchain?

**Comment by EndymionJkb** [comment]
Not that I'm aware of. Not sure there's a value add there anyway vs
centralized; most of these are mobile apps, with small monetary
transactions.

I suppose you could write clever contracts to vote on the tip,
automatically distribute funds, enforce review requirements, etc., but it
would work just fine centralized.

Biggest problem is licensing for food preparation. This might have to wait
until we have automated drone delivery from restaurants. That would be
cool: flash mob catering - when you get a quorum at a location and people
vote to have a meal; food comes to the door, or drops into your lap from
the sky :)

**Comment by fastdraw07** [comment]
@EndymionJkb I don't think a P2P approach will raise costs; just the opposite. If a centralized AirBnB for dinner can get funding with all their administration costs, then a decentralized p2p version will work better. The whole point of the blockchain is to show that government regulation and licensing aren't needed. Smart contracts and the blockchain handle all that.

**Comment by EndymionJkb** [comment]
All true, and I hope we can move toward that. You might get sick at a
backyard barbecue, after all. It's not so much that the government
genuinely cares about protecting people - they just want their cut in
licensing fees and taxes. Like the Puritans who worry that someone,
somewhere, might be having fun, the government worries that someone,
somewhere, might be making money without giving them a piece of it.

So perhaps they could be paid off somehow, like an initial "hosting" fee
that would trigger an inspection and grant a special kind of license. Food
licenses now are geared toward commercial kitchens; there's no way a
regular home kitchen would pass without a major remodel that would defeat
the purpose.

Or maybe the laws could be circumvented by not charging directly for meals
- the government is after their cut of commercial transactions; if there's
no money, there's nothing to take a cut of. Then it would have to be ad
funded though, which is a much harder revenue model.

Perhaps it could be tips only (e.g., crypto to a certain address), and
contracts could track and characterize tipping behavior and influence who
the hosts invited. Guests could stake a deposit to attend a meal, then
signal their tip to be released to the host.

Or maybe people could pay the health department to get licensed as "safety
inspectors," and the contract would require one "inspector" at every meal,
who'd make the fees back through free meals. That way lip service is paid
to safety, the government gets its cut, and hosts are allowed to profit
from the other guests.

It would depend on how aggressive the health department was wherever you
wanted to try it. San Francisco or New York or any big city would probably
be inflexible, to say the least. And a small town with a compliant
bureaucracy might be too small for a valid test.

This is a tough one. It's also brittle, because if God forbid somebody did
get sick or have a problem, it would probably be over. Remember Chi Chi's?
A long-established, nationwide brick and mortar restaurant chain was gone
in a few months after a hepatitis scare - and they weren't even shut down
by the government. They simply lost too much business.


On Jul 14, 2017 12:53 PM, "fastdraw07" <notifications@github.com> wrote:

@EndymionJkb <https://github.com/endymionjkb> I don't think a P2P approach
will raise costs; just the opposite. If a centralized AirBnB for dinner can
get funding with all their administration costs, then a decentralized p2p
version will work better. The whole point of the blockchain is to show that
government regulation and licensing aren't needed. Smart contracts and the
blockchain handle all that.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/district0x/district-proposals/issues/28#issuecomment-315409867>,
or mute the thread

**Comment by T-100** [comment]
Margins on restaurants are paper-thin.  Regulations exist for a reason: food safety.  This idea has been tried, and the concept is essentially flawed from the beginning.  Did you talk with anyone in hospitality operations before moving forward with this idea?  I've got 25 years in restaurant ops, and this idea is laughable.

**Comment by AngeloAdam** [comment]
please send me a direct message on district slack regarding your proposal. @angelo

**Comment by Yursay** [comment]
great idea !

**Comment by Bradymck** [comment]
Hey there, 

I'm not sure if you're aware but we recently launched a new bounty to migrate your proposals to the actual District Registry: https://registry.district0x.io/

We're replacing the old voting app with the registry. Let me know if you need help but I would love to see you migrate this over so you can claim your DNT. 

It does take a 10,000 DNT submit to submit your proposal but this gives you an extra 2000 you can stake in the registry beyond the deposit amount. 

Hit me up on Telegram or Discord if you need help or have questions. 

Telegram: https://t.me/district0x/75217
Discord: https://discord.gg/P9RQejv

PS, please excuse the canned response. I am encouraging everyone here to start migrating so they can claim their 12000 DNT.
